packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -106,8 +109,13 @@ const styles = {
             box-sizing: border-box;
         }
     `,
-    container: css`
+    containerOverrides: css`
         position: relative;
+        width: 100%;
+        max-width: 1300px;
+        margin: 0 auto;
+    `,
+    container: css`
```

Comment:
I copied this across from the current `ElectionAuMomentBanner`. This is needed so that the ac pop stays inline with the banner text. Without this, the overlay is positioned relative to the main banner container, which can be really big, but the content will be at most 1300px wide.

with: 
![Screenshot 2022-05-13 at 12 40 21](https://user-images.githubusercontent.com/17720442/168275958-148f1080-8189-4948-9a4e-8757849b3ad3.png)

without:
![Screenshot 2022-05-13 at 12 40 34](https://user-images.githubusercontent.com/17720442/168275969-16db0278-43c2-4948-8aff-19395648bf83.png)